lib/puppet/provider/iis_webconfigurationproperty/iis_webconfigurationproperty.rb

```diff
@@ -0,0 +1,52 @@
+Puppet::Type.type(:iis_webconfigurationproperty).provide(:iis_webconfigurationproperty) do
```

Comment:
We already have some discussion about this type going. Will have to review further